src/convert.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { convertSelicEntry } from "./convert";
```

Comment:
Try matching the file name with the function/class. Don't forget to change this test file name as well. 😉 

```suggestion
import { convertSelicEntry } from "./convertSelicEntry";
```